data/config.txt
0 /Users/mjl/Documents/Programming/python/kd/63.jpg 360 280
0 /Users/mjl/Documents/Programming/python/kd/77.jpg 680 375
0 /Users/mjl/Documents/Programming/python/kd/88.jpg 596 400
0 /Users/mjl/Documents/Programming/python/kd/89.jpg 900 525
0 /Users/mjl/Documents/Programming/python/kd/76.jpg 294 165
0 /Users/mjl/Documents/Programming/python/kd/62.jpg 500 375
0 /Users/mjl/Documents/Programming/python/kd/74.jpg 600 473
0 /Users/mjl/Documents/Programming/python/kd/48.jpg 549 315
0 /Users/mjl/Documents/Programming/python/kd/149.jpg 846 437
0 /Users/mjl/Documents/Programming/python/kd/148.jpg 750 1000
0 /Users/mjl/Documents/Programming/python/kd/49.jpg 500 322
0 /Users/mjl/Documents/Programming/python/kd/61.jpg 480 360
0 /Users/mjl/Documents/Programming/python/kd/75.jpg 517 297
0 /Users/mjl/Documents/Programming/python/kd/59.jpg 880 510
0 /Users/mjl/Documents/Programming/python/kd/71.jpg 1000 1504
0 /Users/mjl/Documents/Programming/python/kd/65.jpg 612 816
0 /Users/mjl/Documents/Programming/python/kd/70.jpg 240 159
0 /Users/mjl/Documents/Programming/python/kd/58.jpg 1536 2048
0 /Users/mjl/Documents/Programming/python/kd/8.jpg 869 479
0 /Users/mjl/Documents/Programming/python/kd/66.jpg 680 536
0 /Users/mjl/Documents/Programming/python/kd/72.jpg 680 510
0 /Users/mjl/Documents/Programming/python/kd/99.jpg 660 484
0 /Users/mjl/Documents/Programming/python/kd/98.jpg 500 351
0 /Users/mjl/Documents/Programming/python/kd/73.jpg 461 401
0 /Users/mjl/Documents/Programming/python/kd/67.jpg 688 378
0 /Users/mjl/Documents/Programming/python/kd/9.jpg 800 640
0 /Users/mjl/Documents/Programming/python/kd/14.jpg 500 336
0 /Users/mjl/Documents/Programming/python/kd/28.jpg 500 375
0 /Users/mjl/Documents/Programming/python/kd/129.jpg 750 1312
0 /Users/mjl/Documents/Programming/python/kd/101.jpg 602 444
0 /Users/mjl/Documents/Programming/python/kd/115.jpg 560 411
0 /Users/mjl/Documents/Programming/python/kd/114.jpg 80 65
0 /Users/mjl/Documents/Programming/python/kd/100.jpg 680 375
0 /Users/mjl/Documents/Programming/python/kd/128.jpg 640 483
0 /Users/mjl/Documents/Programming/python/kd/29.jpg 400 533
0 /Users/mjl/Documents/Programming/python/kd/17.jpg 620 460
0 /Users/mjl/Documents/Programming/python/kd/116.jpg 545 265
0 /Users/mjl/Documents/Programming/python/kd/102.jpg 350 493
0 /Users/mjl/Documents/Programming/python/kd/103.jpg 351 413
0 /Users/mjl/Documents/Programming/python/kd/117.jpg 1068 663
0 /Users/mjl/Documents/Programming/python/kd/16.jpg 503 304
0 /Users/mjl/Documents/Programming/python/kd/12.jpg 840 645
0 /Users/mjl/Documents/Programming/python/kd/113.jpg 600 359
0 /Users/mjl/Documents/Programming/python/kd/107.jpg 310 310
0 /Users/mjl/Documents/Programming/python/kd/112.jpg 869 506
0 /Users/mjl/Documents/Programming/python/kd/13.jpg 1024 768
0 /Users/mjl/Documents/Programming/python/kd/39.jpg 720 960
0 /Users/mjl/Documents/Programming/python/kd/11.jpg 1068 663
0 /Users/mjl/Documents/Programming/python/kd/104.jpg 500 214
0 /Users/mjl/Documents/Programming/python/kd/110.jpg 800 536
0 /Users/mjl/Documents/Programming/python/kd/138.jpg 500 374
0 /Users/mjl/Documents/Programming/python/kd/139.jpg 680 413
0 /Users/mjl/Documents/Programming/python/kd/111.jpg 808 591
0 /Users/mjl/Documents/Programming/python/kd/10.jpg 644 349
0 /Users/mjl/Documents/Programming/python/kd/38.jpg 680 376
0 /Users/mjl/Documents/Programming/python/kd/21.jpg 200 150
0 /Users/mjl/Documents/Programming/python/kd/35.jpg 680 376
0 /Users/mjl/Documents/Programming/python/kd/108.jpg 667 500
0 /Users/mjl/Documents/Programming/python/kd/134.jpg 1100 611
0 /Users/mjl/Documents/Programming/python/kd/135.jpg 120 80
0 /Users/mjl/Documents/Programming/python/kd/121.jpg 550 318
0 /Users/mjl/Documents/Programming/python/kd/109.jpg 1000 750
0 /Users/mjl/Documents/Programming/python/kd/34.jpg 960 1280
0 /Users/mjl/Documents/Programming/python/kd/20.jpg 700 405
0 /Users/mjl/Documents/Programming/python/kd/22.jpg 435 330
0 /Users/mjl/Documents/Programming/python/kd/137.jpg 550 412
0 /Users/mjl/Documents/Programming/python/kd/123.jpg 205 116
0 /Users/mjl/Documents/Programming/python/kd/122.jpg 500 304
0 /Users/mjl/Documents/Programming/python/kd/136.jpg 480 346
0 /Users/mjl/Documents/Programming/python/kd/23.jpg 700 390
0 /Users/mjl/Documents/Programming/python/kd/37.jpg 580 431
0 /Users/mjl/Documents/Programming/python/kd/33.jpg 800 600
0 /Users/mjl/Documents/Programming/python/kd/27.jpg 310 310
0 /Users/mjl/Documents/Programming/python/kd/132.jpg 1698 1190
0 /Users/mjl/Documents/Programming/python/kd/126.jpg 859 474
0 /Users/mjl/Documents/Programming/python/kd/127.jpg 859 474
0 /Users/mjl/Documents/Programming/python/kd/133.jpg 600 454
0 /Users/mjl/Documents/Programming/python/kd/26.jpg 300 451
0 /Users/mjl/Documents/Programming/python/kd/32.jpg 640 480
0 /Users/mjl/Documents/Programming/python/kd/18.jpg 398 432
0 /Users/mjl/Documents/Programming/python/kd/24.jpg 680 375
0 /Users/mjl/Documents/Programming/python/kd/30.jpg 1174 662
0 /Users/mjl/Documents/Programming/python/kd/125.jpg 680 414
0 /Users/mjl/Documents/Programming/python/kd/119.jpg 420 264
0 /Users/mjl/Documents/Programming/python/kd/118.jpg 680 375
0 /Users/mjl/Documents/Programming/python/kd/130.jpg 680 378
0 /Users/mjl/Documents/Programming/python/kd/124.jpg 280 209
0 /Users/mjl/Documents/Programming/python/kd/31.jpg 260 230
0 /Users/mjl/Documents/Programming/python/kd/25.jpg 1153 720
0 /Users/mjl/Documents/Programming/python/kd/19.jpg 580 318
0 /Users/mjl/Documents/Programming/python/kd/42.jpg 640 323
0 /Users/mjl/Documents/Programming/python/kd/4.jpg 398 432
0 /Users/mjl/Documents/Programming/python/kd/56.jpg 870 503
0 /Users/mjl/Documents/Programming/python/kd/81.jpg 240 180
0 /Users/mjl/Documents/Programming/python/kd/95.jpg 839 604
0 /Users/mjl/Documents/Programming/python/kd/142.jpg 443 453
0 /Users/mjl/Documents/Programming/python/kd/94.jpg 612 816
0 /Users/mjl/Documents/Programming/python/kd/80.jpg 405 320
0 /Users/mjl/Documents/Programming/python/kd/5.jpg 667 500
0 /Users/mjl/Documents/Programming/python/kd/43.jpg 440 330
0 /Users/mjl/Documents/Programming/python/kd/7.jpg 892 596
0 /Users/mjl/Documents/Programming/python/kd/41.jpg 680 376
0 /Users/mjl/Documents/Programming/python/kd/69.jpg 1024 760
0 /Users/mjl/Documents/Programming/python/kd/96.jpg 552 367
0 /Users/mjl/Documents/Programming/python/kd/82.jpg 550 411
0 /Users/mjl/Documents/Programming/python/kd/140.jpg 680 378
0 /Users/mjl/Documents/Programming/python/kd/141.jpg 490 280
0 /Users/mjl/Documents/Programming/python/kd/83.jpg 2816 1880
0 /Users/mjl/Documents/Programming/python/kd/97.jpg 800 480
0 /Users/mjl/Documents/Programming/python/kd/68.jpg 720 848
0 /Users/mjl/Documents/Programming/python/kd/40.jpg 680 373
0 /Users/mjl/Documents/Programming/python/kd/54.jpg 600 331
0 /Users/mjl/Documents/Programming/python/kd/6.jpg 1440 800
0 /Users/mjl/Documents/Programming/python/kd/78.jpg 680 389
0 /Users/mjl/Documents/Programming/python/kd/2.jpg 500 322
0 /Users/mjl/Documents/Programming/python/kd/50.jpg 347 400
0 /Users/mjl/Documents/Programming/python/kd/44.jpg 360 640
0 /Users/mjl/Documents/Programming/python/kd/145.jpg 1024 760
0 /Users/mjl/Documents/Programming/python/kd/87.jpg 2717 1500
0 /Users/mjl/Documents/Programming/python/kd/86.jpg 492 324
0 /Users/mjl/Documents/Programming/python/kd/92.jpg 563 364
0 /Users/mjl/Documents/Programming/python/kd/144.jpg 869 555
0 /Users/mjl/Documents/Programming/python/kd/150.jpg 960 720
0 /Users/mjl/Documents/Programming/python/kd/45.jpg 640 480
0 /Users/mjl/Documents/Programming/python/kd/3.jpg 700 405
0 /Users/mjl/Documents/Programming/python/kd/51.jpg 635 833
0 /Users/mjl/Documents/Programming/python/kd/47.jpg 640 354
0 /Users/mjl/Documents/Programming/python/kd/1.jpg 1024 768
0 /Users/mjl/Documents/Programming/python/kd/146.jpg 864 520
0 /Users/mjl/Documents/Programming/python/kd/84.jpg 869 506
0 /Users/mjl/Documents/Programming/python/kd/90.jpg 1680 1016
0 /Users/mjl/Documents/Programming/python/kd/91.jpg 360 480
0 /Users/mjl/Documents/Programming/python/kd/85.jpg 460 345
0 /Users/mjl/Documents/Programming/python/kd/147.jpg 915 703
0 /Users/mjl/Documents/Programming/python/kd/52.jpg 708 389
0 /Users/mjl/Documents/Programming/python/kd/0.jpg 680 375
0 /Users/mjl/Documents/Programming/python/st/48.jpg 500 363
0 /Users/mjl/Documents/Programming/python/st/49.jpg 1024 761
0 /Users/mjl/Documents/Programming/python/st/8.jpg 710 407
0 /Users/mjl/Documents/Programming/python/st/9.jpg 1296 972
0 /Users/mjl/Documents/Programming/python/st/14.jpg 360 480
0 /Users/mjl/Documents/Programming/python/st/28.jpg 605 816
0 /Users/mjl/Documents/Programming/python/st/29.jpg 1680 1016
0 /Users/mjl/Documents/Programming/python/st/15.jpg 500 215
0 /Users/mjl/Documents/Programming/python/st/17.jpg 665 258
0 /Users/mjl/Documents/Programming/python/st/16.jpg 869 506
0 /Users/mjl/Documents/Programming/python/st/12.jpg 828 457
0 /Users/mjl/Documents/Programming/python/st/39.jpg 540 960
0 /Users/mjl/Documents/Programming/python/st/11.jpg 500 365
0 /Users/mjl/Documents/Programming/python/st/10.jpg 644 349
0 /Users/mjl/Documents/Programming/python/st/38.jpg 500 229
0 /Users/mjl/Documents/Programming/python/st/21.jpg 500 322
0 /Users/mjl/Documents/Programming/python/st/35.jpg 500 251
0 /Users/mjl/Documents/Programming/python/st/34.jpg 582 446
0 /Users/mjl/Documents/Programming/python/st/20.jpg 489 474
0 /Users/mjl/Documents/Programming/python/st/36.jpg 936 1664
0 /Users/mjl/Documents/Programming/python/st/22.jpg 500 447
0 /Users/mjl/Documents/Programming/python/st/23.jpg 500 331
0 /Users/mjl/Documents/Programming/python/st/27.jpg 803 471
0 /Users/mjl/Documents/Programming/python/st/26.jpg 500 285
0 /Users/mjl/Documents/Programming/python/st/32.jpg 310 310
0 /Users/mjl/Documents/Programming/python/st/18.jpg 1024 768
0 /Users/mjl/Documents/Programming/python/st/24.jpg 260 230
0 /Users/mjl/Documents/Programming/python/st/30.jpg 500 388
0 /Users/mjl/Documents/Programming/python/st/31.jpg 794 706
0 /Users/mjl/Documents/Programming/python/st/25.jpg 500 283
0 /Users/mjl/Documents/Programming/python/st/19.jpg 1421 320
0 /Users/mjl/Documents/Programming/python/st/42.jpg 500 405
0 /Users/mjl/Documents/Programming/python/st/4.jpg 450 254
0 /Users/mjl/Documents/Programming/python/st/5.jpg 500 227
0 /Users/mjl/Documents/Programming/python/st/43.jpg 500 309
0 /Users/mjl/Documents/Programming/python/st/41.jpg 588 552
0 /Users/mjl/Documents/Programming/python/st/40.jpg 750 1330
0 /Users/mjl/Documents/Programming/python/st/6.jpg 600 450
0 /Users/mjl/Documents/Programming/python/st/2.jpg 869 506
0 /Users/mjl/Documents/Programming/python/st/50.jpg 240 180
0 /Users/mjl/Documents/Programming/python/st/44.jpg 338 600
0 /Users/mjl/Documents/Programming/python/st/45.jpg 539 543
0 /Users/mjl/Documents/Programming/python/st/3.jpg 605 816
0 /Users/mjl/Documents/Programming/python/st/51.jpg 500 394
0 /Users/mjl/Documents/Programming/python/st/47.jpg 1334 461
0 /Users/mjl/Documents/Programming/python/st/1.jpg 265 148
0 /Users/mjl/Documents/Programming/python/st/46.jpg 750 1312
0 /Users/mjl/Documents/Programming/python/sf/8.jpg 800 800
0 /Users/mjl/Documents/Programming/python/sf/9.jpg 816 612
0 /Users/mjl/Documents/Programming/python/sf/14.jpg 1024 760
0 /Users/mjl/Documents/Programming/python/sf/28.jpg 682 448
0 /Users/mjl/Documents/Programming/python/sf/29.jpg 294 320
0 /Users/mjl/Documents/Programming/python/sf/15.jpg 440 331
0 /Users/mjl/Documents/Programming/python/sf/17.jpg 955 523
0 /Users/mjl/Documents/Programming/python/sf/16.jpg 600 343
0 /Users/mjl/Documents/Programming/python/sf/12.jpg 800 676
0 /Users/mjl/Documents/Programming/python/sf/13.jpg 816 555
0 /Users/mjl/Documents/Programming/python/sf/39.jpg 500 282
0 /Users/mjl/Documents/Programming/python/sf/11.jpg 270 480
0 /Users/mjl/Documents/Programming/python/sf/10.jpg 720 399
0 /Users/mjl/Documents/Programming/python/sf/38.jpg 480 245
0 /Users/mjl/Documents/Programming/python/sf/21.jpg 690 530
0 /Users/mjl/Documents/Programming/python/sf/35.jpg 300 471
0 /Users/mjl/Documents/Programming/python/sf/34.jpg 668 546
0 /Users/mjl/Documents/Programming/python/sf/20.jpg 698 404
0 /Users/mjl/Documents/Programming/python/sf/36.jpg 320 568
0 /Users/mjl/Documents/Programming/python/sf/22.jpg 500 267
0 /Users/mjl/Documents/Programming/python/sf/23.jpg 816 555
0 /Users/mjl/Documents/Programming/python/sf/37.jpg 580 326
0 /Users/mjl/Documents/Programming/python/sf/33.jpg 794 190
0 /Users/mjl/Documents/Programming/python/sf/27.jpg 600 454
0 /Users/mjl/Documents/Programming/python/sf/26.jpg 466 295
0 /Users/mjl/Documents/Programming/python/sf/32.jpg 750 502
0 /Users/mjl/Documents/Programming/python/sf/18.jpg 500 315
0 /Users/mjl/Documents/Programming/python/sf/24.jpg 369 268
0 /Users/mjl/Documents/Programming/python/sf/30.jpg 500 226
0 /Users/mjl/Documents/Programming/python/sf/31.jpg 959 590
0 /Users/mjl/Documents/Programming/python/sf/25.jpg 1800 995
0 /Users/mjl/Documents/Programming/python/sf/19.jpg 2816 1880
0 /Users/mjl/Documents/Programming/python/sf/42.jpg 425 319
0 /Users/mjl/Documents/Programming/python/sf/4.jpg 800 800
0 /Users/mjl/Documents/Programming/python/sf/5.jpg 550 380
0 /Users/mjl/Documents/Programming/python/sf/43.jpg 310 310
0 /Users/mjl/Documents/Programming/python/sf/7.jpg 688 378
0 /Users/mjl/Documents/Programming/python/sf/41.jpg 240 180
0 /Users/mjl/Documents/Programming/python/sf/40.jpg 800 600
0 /Users/mjl/Documents/Programming/python/sf/6.jpg 800 800
0 /Users/mjl/Documents/Programming/python/sf/2.jpg 360 640
0 /Users/mjl/Documents/Programming/python/sf/3.jpg 225 300
0 /Users/mjl/Documents/Programming/python/sf/1.jpg 500 375
0 /Users/mjl/Documents/Programming/python/sf/0.jpg 866 563
0 /Users/mjl/Documents/Programming/python/yt/48.jpg 666 443
0 /Users/mjl/Documents/Programming/python/yt/49.jpg 500 276
0 /Users/mjl/Documents/Programming/python/yt/8.jpg 3072 2304
0 /Users/mjl/Documents/Programming/python/yt/9.jpg 720 399
0 /Users/mjl/Documents/Programming/python/yt/14.jpg 500 333
0 /Users/mjl/Documents/Programming/python/yt/28.jpg 667 500
0 /Users/mjl/Documents/Programming/python/yt/29.jpg 310 310
0 /Users/mjl/Documents/Programming/python/yt/15.jpg 480 640
0 /Users/mjl/Documents/Programming/python/yt/17.jpg 440 399
0 /Users/mjl/Documents/Programming/python/yt/16.jpg 500 385
0 /Users/mjl/Documents/Programming/python/yt/12.jpg 867 473
0 /Users/mjl/Documents/Programming/python/yt/13.jpg 960 1280
0 /Users/mjl/Documents/Programming/python/yt/39.jpg 500 207
0 /Users/mjl/Documents/Programming/python/yt/11.jpg 300 300
0 /Users/mjl/Documents/Programming/python/yt/10.jpg 667 500
0 /Users/mjl/Documents/Programming/python/yt/38.jpg 500 259
0 /Users/mjl/Documents/Programming/python/yt/21.jpg 500 307
0 /Users/mjl/Documents/Programming/python/yt/35.jpg 720 960
0 /Users/mjl/Documents/Programming/python/yt/34.jpg 802 544
0 /Users/mjl/Documents/Programming/python/yt/20.jpg 644 349
0 /Users/mjl/Documents/Programming/python/yt/36.jpg 750 1312
0 /Users/mjl/Documents/Programming/python/yt/22.jpg 750 1330
0 /Users/mjl/Documents/Programming/python/yt/23.jpg 803 471
0 /Users/mjl/Documents/Programming/python/yt/33.jpg 1680 1016
0 /Users/mjl/Documents/Programming/python/yt/27.jpg 500 301
0 /Users/mjl/Documents/Programming/python/yt/26.jpg 400 411
0 /Users/mjl/Documents/Programming/python/yt/32.jpg 1536 2048
0 /Users/mjl/Documents/Programming/python/yt/18.jpg 600 343
0 /Users/mjl/Documents/Programming/python/yt/24.jpg 500 369
0 /Users/mjl/Documents/Programming/python/yt/30.jpg 338 600
0 /Users/mjl/Documents/Programming/python/yt/31.jpg 260 230
0 /Users/mjl/Documents/Programming/python/yt/25.jpg 1276 690
0 /Users/mjl/Documents/Programming/python/yt/19.jpg 710 407
0 /Users/mjl/Documents/Programming/python/yt/42.jpg 600 454
0 /Users/mjl/Documents/Programming/python/yt/4.jpg 1000 1504
0 /Users/mjl/Documents/Programming/python/yt/5.jpg 667 500
0 /Users/mjl/Documents/Programming/python/yt/43.jpg 500 257
0 /Users/mjl/Documents/Programming/python/yt/7.jpg 640 480
0 /Users/mjl/Documents/Programming/python/yt/41.jpg 500 394
0 /Users/mjl/Documents/Programming/python/yt/40.jpg 140 140
0 /Users/mjl/Documents/Programming/python/yt/6.jpg 263 210
0 /Users/mjl/Documents/Programming/python/yt/2.jpg 1280 959
0 /Users/mjl/Documents/Programming/python/yt/50.jpg 1024 565
0 /Users/mjl/Documents/Programming/python/yt/44.jpg 1025 595
0 /Users/mjl/Documents/Programming/python/yt/45.jpg 1424 323
0 /Users/mjl/Documents/Programming/python/yt/3.jpg 657 369
0 /Users/mjl/Documents/Programming/python/yt/51.jpg 500 182
0 /Users/mjl/Documents/Programming/python/yt/47.jpg 500 207
0 /Users/mjl/Documents/Programming/python/yt/53.jpg 500 501
0 /Users/mjl/Documents/Programming/python/yt/1.jpg 492 324
0 /Users/mjl/Documents/Programming/python/yt/52.jpg 413 616
0 /Users/mjl/Documents/Programming/python/yt/0.jpg 800 800
0 /Users/mjl/Documents/Programming/python/yt/46.jpg 450 257
0 /Users/mjl/Documents/Programming/python/zt/63.jpg 500 888
0 /Users/mjl/Documents/Programming/python/zt/77.jpg 658 872
0 /Users/mjl/Documents/Programming/python/zt/88.jpg 853 572
0 /Users/mjl/Documents/Programming/python/zt/89.jpg 967 987
0 /Users/mjl/Documents/Programming/python/zt/76.jpg 1024 625
0 /Users/mjl/Documents/Programming/python/zt/62.jpg 535 741
0 /Users/mjl/Documents/Programming/python/zt/74.jpg 1130 259
0 /Users/mjl/Documents/Programming/python/zt/60.jpg 450 337
0 /Users/mjl/Documents/Programming/python/zt/48.jpg 500 215
0 /Users/mjl/Documents/Programming/python/zt/49.jpg 967 987
0 /Users/mjl/Documents/Programming/python/zt/61.jpg 1024 724
0 /Users/mjl/Documents/Programming/python/zt/75.jpg 340 573
0 /Users/mjl/Documents/Programming/python/zt/59.jpg 506 900
0 /Users/mjl/Documents/Programming/python/zt/71.jpg 600 454
0 /Users/mjl/Documents/Programming/python/zt/65.jpg 500 304
0 /Users/mjl/Documents/Programming/python/zt/64.jpg 1024 590
0 /Users/mjl/Documents/Programming/python/zt/70.jpg 347 315
0 /Users/mjl/Documents/Programming/python/zt/58.jpg 310 310
0 /Users/mjl/Documents/Programming/python/zt/8.jpg 500 322
0 /Users/mjl/Documents/Programming/python/zt/66.jpg 580 374
0 /Users/mjl/Documents/Programming/python/zt/72.jpg 725 987
0 /Users/mjl/Documents/Programming/python/zt/99.jpg 534 246
0 /Users/mjl/Documents/Programming/python/zt/98.jpg 500 247
0 /Users/mjl/Documents/Programming/python/zt/73.jpg 568 542
0 /Users/mjl/Documents/Programming/python/zt/67.jpg 300 451
0 /Users/mjl/Documents/Programming/python/zt/9.jpg 552 367
0 /Users/mjl/Documents/Programming/python/zt/14.jpg 687 510

python/config.py

def data(classes, trainPath, validPath, namesPath):
    dataString = \
'''
classes= %d
train  = %s
valid  = %s
names  = %s
backup = backup
''' \
    %(classes, trainPath, validPath, namesPath)
    return dataString

def cfg(classes, batchs):
    cfgString = \
'''
[net]
batch=64
subdivisions=16
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = {maxBatchs}
policy=steps
steps={maxBatchs_2},{maxBatchs_3}
scales=.1,.1



[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

# Downsample

[convolutional]
batch_normalize=1
filters=64
size=3
stride=2
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=32
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

# Downsample

[convolutional]
batch_normalize=1
filters=128
size=3
stride=2
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=64
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

[convolutional]
batch_normalize=1
filters=64
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

# Downsample

[convolutional]
batch_normalize=1
filters=256
size=3
stride=2
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear


[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

# Downsample

[convolutional]
batch_normalize=1
filters=512
size=3
stride=2
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear


[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear


[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear


[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear


[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear


[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

# Downsample

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=2
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

[convolutional]
batch_normalize=1
filters=512
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

[convolutional]
batch_normalize=1
filters=512
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

[convolutional]
batch_normalize=1
filters=512
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

[shortcut]
from=-3
activation=linear

######################

[convolutional]
batch_normalize=1
filters=512
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters={filters}
activation=linear

[yolo]
mask = 6,7,8
anchors = 10,13,  16,30,  33,23,  30,61,  62,45,  59,119,  116,90,  156,198,  373,326
classes={classes}
num=9
jitter=.3
ignore_thresh = .5
truth_thresh = 1
random=1

[route]
layers = -4

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[upsample]
stride=2

[route]
layers = -1, 61



[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=512
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=512
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=512
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters={filters}
activation=linear

[yolo]
mask = 3,4,5
anchors = 10,13,  16,30,  33,23,  30,61,  62,45,  59,119,  116,90,  156,198,  373,326
classes={classes}
num=9
jitter=.3
ignore_thresh = .5
truth_thresh = 1
random=1

[route]
layers = -4

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[upsample]
stride=2

[route]
layers = -1, 36



[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=256
activation=leaky

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=256
activation=leaky

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=256
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters={filters}
activation=linear

[yolo]
mask = 0,1,2
anchors = 10,13,  16,30,  33,23,  30,61,  62,45,  59,119,  116,90,  156,198,  373,326
classes={classes}
num=9
jitter=.3
ignore_thresh = .5
truth_thresh = 1
random=1


''' \
    .format(maxBatchs=batchs, maxBatchs_2=int(batchs/2), maxBatchs_3=int(3*batchs/4), filters=(3*(5+classes)),classes=classes)
    return cfgString
